assets/src/pages/home/HomePage.tsx

```diff
@@ -103,6 +103,7 @@ export const HomePage: FC = () => {
                 cameraAutostart.setCameraAutostart(autostartCameraAndMicInput);
               }}
               to={`/room/${roomIdInput}`}
+              id="join"
```

Comment:
I needed id to somehow click the link. Otherwise, CSS selector is the `className`